### PR TITLE
[DF] Add two simple tutorials in C++ and Python

### PR DESCRIPTION
**Pull request by etejedor**
To be linked from the ROOT manual in the "First Steps" section.

### CODE REVIEW

File: tutorials/dataframe/df000_simple.C [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+   // Create a data frame with 100 rows
+   ROOT::RDataFrame rdf(100);
+
+   // Define a new column `x` that contains random numbers
+   auto rdf_x = rdf.Define("x", [](){ return gRandom->Rndm(); });
+
+   // Create a histogram from `x` and draw it
+   auto h = rdf_x.Histo1D("x");
+   h->DrawClone();
```
Comment (Axel-Naumann) [inline tutorials/dataframe/df000_simple.C:25]:
```suggestion

   /// At the end of this function, the histogram pointed to by `h` will be deleted.
   /// Draw a copy of the histogram object instead:
   h->DrawClone();
```

**Review by jalopezg-r00t — APPROVED** [review]
LGTM.

File: tutorials/dataframe/df000_simple.C [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// Simple RDataFrame example in C++.
+///
+/// This tutorial shows a minimal example of RDataFrame that creates an
+/// empty-source data frame, adds a new column `x` with random numbers
+/// and finally creates and draws a histogram for `x`.
```
Comment (Axel-Naumann) [inline tutorials/dataframe/df000_simple.C:7]: I don't find "empty-source" self-explanatory. Maybe
 
```suggestion
/// This tutorial shows a minimal example of RDataFrame. It starts without input
/// data, generates a new column `x` with random numbers, and finally draws
/// a histogram for `x`.
```

**Review by Axel-Naumann — APPROVED** [review]
Nice! And see comments

**Comment by etejedor** [comment]
Thanks @Axel-Naumann , comments applied. Merging so that we get a link tomorrow for the First Steps section.

**Pull request merged**